Lib/code.py

```diff
@@ -94,7 +94,7 @@ def runcode(self, code):
         except:
             self.showtraceback()
 
-    def showsyntaxerror(self, filename=None, **kwargs):
+    def showsyntaxerror(self, filename=None):
```

Comment:
This is potentially a breaking change. `**kwargs` was added in #119318, but it was not documented, and is not supported in subclasses, so we can consider it a temporary hack. I am fine with this change.

Reply:
makes sense to me